src/components/project/form-row.vue

```diff
@@ -0,0 +1,111 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <tr class="project-form-row">
+    <!--todo: come back and figure out all the permissions-->
+    <td class="name">
+      <link-if-can :to="primaryFormPath(form)">
+        {{ form.nameOrId() }}
+      </link-if-can>
+    </td>
+    <td class="review-state">
+      {{ $n(form.reviewState.received) }}<span class="icon-dot-circle-o"></span>
+    </td>
+    <td class="review-state">
+      {{ $n(form.reviewState.hasIssues) }}<span class="icon-comments"></span>
+    </td>
+    <td class="review-state">
+      {{ $n(form.reviewState.edited) }}<span class="icon-pencil"></span>
+    </td>
+    <td class="last-submission">
+      <div v-if="form.lastSubmission != null">
+        <router-link :to="submissionsPath">
+          <date-time :iso="form.lastSubmission" relative="past"/>
```

Comment:
I think the `blank` prop could be specified here. (You can grab that from `ProjectRow`. I think that message will now be used in multiple components, so should probably be moved to `en.json5`.)